### DISCUSSION TITLE
Error: The "serverless-plugin-chrome" plugin only supports the Node.js 6.10 or 8.10

### DISCUSSION THREAD

**Discussion by karim-haned** [serverless/serverless#12285] [General]
Whenever I attempt to deploy my function, I consistently encounter errors related to incompatible Node.js versions.

1. In my serverless yml file , if runtime is nodejs18.x : this error occurs : Error: The "serverless-plugin-chrome" plugin only supports the Node.js 6.10 or 8.10
2. If I downgrade to version 8.10 , this error occurs during chrome injection : "The runtime parameter of nodejs8.10 is no longer supported for creating or updating AWS Lambda functions.
![image](https://github.com/serverless/serverless/assets/68304005/eb0b9783-0282-46e2-a87c-8e3c115fc57a)

**Comment by muhzak** [comment]
@karim-haned  did you figure this out?
